pr2_moveit_plugins/package.xml

```diff
@@ -1,10 +1,11 @@
-<package>
+<package format="2">
   <name>pr2_moveit_plugins</name>
   <version>0.6.3</version>
   <description>PR2 specific plugins for MoveIt</description>
   <author email="isucan@willowgarage.com">Ioan Sucan</author>
   <author email="sachinc@willowgarage.com">Sachin Chitta</author>
 
+  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
   <maintainer email="isucan@willowgarage.com">Ioan Sucan</maintainer>
   <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
```

Comment:
May not be a big deal but we probably want to remove these 2 maintainers.

Reply:
Resolved